packages/ui/FloatingLayer/tests/FloatingLayer-specs.js

```diff
@@ -1,31 +1,34 @@
-import {mount} from 'enzyme';
+import {render, screen} from '@testing-library/react';
+
 import {FloatingLayerBase} from '../FloatingLayer';
 import FloatingLayerDecorator from '../FloatingLayerDecorator';
 
 describe('FloatingLayer Specs', () => {
 	const Root = FloatingLayerDecorator('div');
 
 	test('should not render if FloatingLayer is not open', () => {
-		const subject = mount(
-			<Root>
+		render(
+			<Root data-testid="rootElement">
 				<FloatingLayerBase><p>Hi</p></FloatingLayerBase>
 			</Root>
 		);
 
-		const expected = null;
-		const actual = subject.find('FloatingLayer').instance().node;
-		expect(actual).toBe(expected);
+		const expected = 0;
+		const floatingLayerContainer = screen.getByTestId('rootElement').nextElementSibling.children.length;
+
+		expect(floatingLayerContainer).toBe(expected);
```

Comment:
How about adding `data-testid` to `FloatingLayerBase` and doing below?
You may need to add `import '@testing-library/jest-dom';` to the top.

```suggestion
		const node = screen.getByTestId('rootElement');

		expect(node).not.toBeInTheDocument();
```